typings/organisation.d.ts

```diff
@@ -0,0 +1,8 @@
+import { DefaultElements } from './defaultElements'
+import { MetaSys, MetaSysProps } from './meta'
+
+export interface OrganisationProp {
```

Comment:
It would be nice to change it to the American English format to stay consistent.
`Organisation` -> `Organization`

Reply:
agree and changed